### PR TITLE
[minor_change] Removed "Required" condition for mac attribute

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

File: website/docs/d/destination_of_redirected_traffic.html.markdown [hunk 1]
```diff
@@ -24,16 +33,16 @@ data "aci_destination_of_redirected_traffic" "example" {
...
 
 ## Attribute Reference
 
 - `id` - Attribute id set to the Dn of the Destination of redirected traffic.
-- `annotation` - (Optional) Annotation for object destination of redirected traffic.
+- `annotation` - (Optional) Annotation for the object destination of redirected traffic.
 - `destination` - (Optional) Specifies the description of a policy component.
-- `dest_name` - (Optional) The destination name to which data was exported. This utility creates a summary report containing configuration information, logs and diagnostic data that will help TAC in troubleshooting and resolving a technical issue.
-- `ip2` - (Optional) IP2 for object destination of redirected traffic.
+- `dest_name` - (Optional) The destination name to which data was exported.
```
Comment (lhercot) [inline website/docs/d/destination_of_redirected_traffic.html.markdown:44]: I don't think those descriptions are correct.

File: website/docs/r/destination_of_redirected_traffic.html.markdown [hunk 2]
```diff
@@ -31,14 +39,14 @@ resource "aci_destination_of_redirected_traffic" "example" {
...
-- `service_redirect_policy_dn` - (Required) Distinguished name of parent Service Redirect Policy object.
+- `service_redirect_policy_dn` - (Required) Distinguished name of the parent Service Redirect Policy object.
 - `ip` - (Required) The IP address.
-- `mac` - (Required) The MAC address.
-- `annotation` - (Optional) Annotation for object destination of redirected traffic.
-- `description` - (Optional) Description for object destination of redirected traffic.
-- `dest_name` - (Optional) The destination name to which data was exported. This utility creates a summary report containing configuration information, logs and diagnostic data that will help TAC in troubleshooting and resolving a technical issue.
-- `ip2` - (Optional) IP2 for object destination of redirected traffic. Default value: "0.0.0.0"
-- `name_alias` - (Optional) Name alias for object destination of redirected traffic.
+- `mac` - (Optional) The MAC address. This is a required value for APIC prior to Version 5.2 release. This value can be foregone by enabling IPSLA on APIC Version 5.2 and above due to dynamic mac detection feature.
```
Comment (lhercot) [inline website/docs/r/destination_of_redirected_traffic.html.markdown:44]: Version -> version

File: website/docs/r/destination_of_redirected_traffic.html.markdown [hunk 3]
```diff
@@ -31,14 +39,14 @@ resource "aci_destination_of_redirected_traffic" "example" {
...
+- `annotation` - (Optional) Annotation for the object destination of redirected traffic.
+- `description` - (Optional) Description for the object destination of redirected traffic.
+- `dest_name` - (Optional) The destination name to which the data was exported.
```
Comment (lhercot) [inline website/docs/r/destination_of_redirected_traffic.html.markdown:47]: I don't think this description is clear

**Review by lhercot — CHANGES_REQUESTED** [review]

File: website/docs/d/destination_of_redirected_traffic.html.markdown [hunk 4]
```diff
@@ -24,16 +33,16 @@ data "aci_destination_of_redirected_traffic" "example" {
...
 ## Attribute Reference
 
 - `id` - Attribute id set to the Dn of the Destination of redirected traffic.
-- `annotation` - (Optional) Annotation for object destination of redirected traffic.
+- `annotation` - (Optional) Annotation for the object destination of redirected traffic.
 - `destination` - (Optional) Specifies the description of a policy component.
-- `dest_name` - (Optional) The destination name to which data was exported. This utility creates a summary report containing configuration information, logs and diagnostic data that will help TAC in troubleshooting and resolving a technical issue.
-- `ip2` - (Optional) IP2 for object destination of redirected traffic.
+- `dest_name` - (Optional) The destination name to which data was exported. 
+- `ip2` - (Optional) IP2 for the object destination of redirected traffic.
```
Comment (anvitha-jain) [inline website/docs/d/destination_of_redirected_traffic.html.markdown:45]: IP2 for the object destination of redirected traffic. -> IP2 of the destination of redirected traffic object.

File: website/docs/d/destination_of_redirected_traffic.html.markdown [hunk 5]
```diff
@@ -24,16 +33,16 @@ data "aci_destination_of_redirected_traffic" "example" {
...
 - `mac` - (Optional) The MAC address.
-- `name_alias` - (Optional) Name alias for object destination of redirected traffic.
+- `name_alias` - (Optional) Name alias for the object destination of redirected traffic.
```
Comment (anvitha-jain) [inline website/docs/d/destination_of_redirected_traffic.html.markdown:47]: Name alias for the object destination of redirected traffic. -> Name alias of the destination of redirected traffic object.

File: website/docs/r/destination_of_redirected_traffic.html.markdown [hunk 6]
```diff
@@ -31,14 +39,14 @@ resource "aci_destination_of_redirected_traffic" "example" {
...
 - `ip` - (Required) The IP address.
-- `mac` - (Required) The MAC address.
-- `annotation` - (Optional) Annotation for object destination of redirected traffic.
-- `description` - (Optional) Description for object destination of redirected traffic.
-- `dest_name` - (Optional) The destination name to which data was exported. This utility creates a summary report containing configuration information, logs and diagnostic data that will help TAC in troubleshooting and resolving a technical issue.
-- `ip2` - (Optional) IP2 for object destination of redirected traffic. Default value: "0.0.0.0"
-- `name_alias` - (Optional) Name alias for object destination of redirected traffic.
+- `mac` - (Optional) The MAC address. This is a required value for APIC prior to Version 5.2 release. This value can be foregone by enabling IPSLA on APIC Version 5.2 and above due to dynamic mac detection feature.
+- `annotation` - (Optional) Annotation for the object destination of redirected traffic.
+- `description` - (Optional) Description for the object destination of redirected traffic.
```
Comment (anvitha-jain) [inline website/docs/r/destination_of_redirected_traffic.html.markdown:46]: for the object destination of redirected traffic -> of the destination of redirected traffic object.

**Review by anvitha-jain — CHANGES_REQUESTED** [review]

File: website/docs/d/destination_of_redirected_traffic.html.markdown (hunk 1 above)
Comment (shrsr) [reply website/docs/d/destination_of_redirected_traffic.html.markdown]: Changed them

**Review by lhercot — APPROVED** [review]
LGTM

**Review by anvitha-jain — APPROVED** [review]
LGTM

**Pull request merged**